File: src/main/java/my/maryvk/maryvkweb/repository/RelationChangeRepository.java
package my.maryvk.maryvkweb.repository;

import my.maryvk.maryvkweb.domain.RelationChange;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface RelationChangeRepository extends JpaRepository<RelationChange, Long> {
    List<RelationChange> findAllByOwnerIdOrderByTimeDesc(Integer ownerId);
    List<RelationChange> findAllByOrderByTimeDesc();
}

File: src/main/java/my/maryvk/maryvkweb/repository/RelationChangeRepository.kt
package my.maryvk.maryvkweb.repository

import my.maryvk.maryvkweb.domain.RelationChange
import org.springframework.data.jpa.repository.JpaRepository

interface RelationChangeRepository : JpaRepository<RelationChange, Long> {
    fun findAllByOwnerIdOrderByTimeDesc(ownerId: Int?): List<RelationChange>
    fun findAllByOrderByTimeDesc(): List<RelationChange>
}